charts/nsq/templates/metrics-service.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       protocol: TCP
       name: http-metrics
     {{- if .Values.metrics.service.extraPorts }}
-    {{- include "common.tplvalues.render" (dict "value" .Values.metrics.service.extraPorts "context" $) | nindent 4 }}
+    {{- include "nsq.tplValues.render" (dict "value" .Values.metrics.service.extraPorts "context" $) | nindent 4 }}
```

Comment:
Indeed, "common." is wrong, it should be "nsq."

But, are these references not case-sensitive? Because it's "tplvalues" with a lower-case "v" where it is currently defined (and everywhere it is already correctly referenced with "nsq.").

https://github.com/nsqio/helm-chart/blob/8b93118d8b761e342ffdfdd6ebc7a9c66d270ed2/charts/nsq/templates/_helpers.tpl#L114

https://github.com/nsqio/helm-chart/blob/8b93118d8b761e342ffdfdd6ebc7a9c66d270ed2/charts/nsq/templates/nsqd-statefulset.yaml#L89

Reply:
good catch, casing change is not intended

Reply:
thanks!